src/main/java/com/hipo_delivery_be/App.java

```diff
@@ -0,0 +1,16 @@
+package com.hipo_delivery_be;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@SpringBootApplication
+public class App {
```

Comment:
HipoDeliveryApplication

Reply:
@caominhnhut Updated to HipoDeliveryApplication on new commit